src/sage/rings/valuation/inductive_valuation.py

```diff
@@ -0,0 +1,1526 @@
+# -*- coding: utf-8 -*-
+r"""
+Inductive valuations on polynomial rings
+
+This module provides functionality for inductive valuations, i.e., finite
+chains of :class:`AugmentedValuation`s on top of a :class:`GaussValuation`.
```

Comment:
Does this need to be :class:`sage.rings.valuation.gauss_valuation.GaussValuation` ?